changelog/20453.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ldap: Set default value for `max_page_size` properly
```

Comment:
nit: To follow changelog convention and make it clear the fix is in the auth method

```suggestion
auth/ldap: Set default value for `max_page_size` properly
```